deploy/shellscript/juliaup-init.sh

```diff
@@ -36,9 +36,12 @@ USAGE:
     juliaup-init [FLAGS] [OPTIONS]
 
 FLAGS:
-    -y, --yes               Disable confirmation prompt.
-        --no-add-to-path    Don't add to user PATH environment variable
-    -h, --help              Prints help information
+    -y, --yes                   Disable confirmation prompt.
+        --no-add-to-path        Don't add to user PATH environment variable
```

Comment:
I would try to express this without the negative. Otherwise if one adds values to choose from for option you end up in the situation where you have `--no-add-to-path=false` and then everyone is left scratching their heads trying to decode the double negative. The alternative is to end up with `--add-to-path` and `--no-add-to-path`, but having `--add-to-path=[yes|no|ask]` seems like the saner eventuality (and is how Julia command-line options work).

Reply:
Makes sense, I've renamed it to `--add-to-path` with a default value of `yes`. If the user enter neither `yes` or `no`, the installer prints a helpful error message: `Error: --add-to-path can be either 'yes' or 'no', you entered: 'true'`

I was tempted to make it be able to parse `true`, and `1` for `yes` and so on, but I wanted to keep all these changes minimal.